hie-ide-backend/Haskell/Ide/IdeBackend.hs

```diff
@@ -0,0 +1,167 @@
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE PartialTypeSignatures #-}
+{-# OPTIONS_GHC -fno-warn-partial-type-signatures #-}
+module Haskell.Ide.IdeBackend where
```

Comment:
Good point, I’m terrible when it comes to documentation. I’ll add it later today or tomorrow.